### ISSUE TITLE
I recommend 1MQT instead of AOL as a query log

### ISSUE THREAD

**Issue by lemire** [powturbo/TurboPFor#1]
I recommend you benchmark with the TREC Million Query Track  (1MQT) instead of the AOL query log.

There are a few reasons :
- There are copyright issues with the AOL query log. There is no clean and legal way to acquire it.
- The AOL query log is "dirty". Though this might be more realistic in a sense, it tends to exacerbate indexing issues. 
- For these two reasons, our paper (http://arxiv.org/abs/1401.6399) only includes the 1MQ results, and the last version does not even allude to the AOL query log. But this means that if you use only the AOL query log, you will not be able to compare with our paper. Now, hitting 1000 queries per second with GOV2 on a single core sounds excellent, but it would be much easier to appreciate if we could compare directly with our results (http://arxiv.org/abs/1401.6399).
- Final reason: switching to 1MQT should be trivial. It does not require the production of new software.

Cheers and keep up the good work!

**Comment by powturbo** [comment]
Hi,

thank you for the info and your comments.
I have updated the benchmark section on the front page (README.md) with
my results.
It will be nice, if someone can report the results on a more recent and
powerful CPU.

It is very simple and fast to setup a benchmark:
- clone TurboPFor
- build with: make
- create index: ./idxcr gov2.sorted
- run: ./idxqry gov2.sorted.i 1mq.txt

Idxqry makes 3 passes and reports the results.

Thank you for preparing and providing the datasets for downloading.
If you want, you can write directly to my gmail address.

Cheers.

**Comment by lemire** [comment]
@powturbo 

I plan to study your results more closely this year, including the reproduction of your good results. When I noticed that you were working on intersections, and getting good results, I figured I had to get in touch. 

We might want to chat about possible collaboration if you want. We seem to have common interests. Though friendly competition is productive in generating new ideas; documentation, benchmarking and analysis often benefit from collaboration.

Have you had a look at Trotman's recent work (QMX): http://dl.acm.org/citation.cfm?id=2682870 ? He has some interesting designs.

(Your project was pointed out to me  by @nkurz . I also recall that you commented on my blog http://lemire.me/blog/archives/2014/05/19/decoding-over-4-billion-integers-per-second-in-c/#comment-128448 .)
